src/MailViewer.php

```diff
@@ -53,7 +56,7 @@ public static function find(string $mail)
 
                     if (is_string($dependency) && class_exists($dependency)) {
                         if (isset($eloquentFactory[$dependency])) {
-                            $args[] = factory($dependency)->states($factoryStates)->make();
+                            $args[] = factory($dependency)->states($factoryStates)->create();
```

Comment:
how are the factories persisted to the db if we are not using `DB::commit()` after this? 🤔

Reply:
The factories don't actually have to be persisted in the DB for the view to work. As long as they exist when `return new $mailable(...$args);` is called the view is fine. They are never committed so keeps the database clean of data that was created just for the purpose of rendering the email view.